doc/cpp/hardware.rst

```diff
@@ -249,6 +249,18 @@ The core Arbor library *libarbor* provides an API for:
 
         The number of CPU threads available.
 
+    .. cpp:member:: bool bind_procs
+
+        Try to generate a binding mask for all processes on a node.
```

Comment:
Document/link what a binding mask is, and how it is generated. (Also on Python page)